aquilon/netbox2aquilon.py

```diff
@@ -320,6 +327,10 @@ def _main():
             netbox2aquilon.config['aquilon']['osversion']
         ),
     )
+    parser.add_argument(
```

Comment:
I understand the need for a fallback personality, but I'm not sure I like allowing people to ignore the netbox information and set arbitrary personalities on devices. Perhaps a lookup table of mappings of netbox-derived mappings to the "legacy" aquilon personality would be better?

Reply:
I don't think that's going to be possible for a while.
But if there was a place to put a mapping it'd be in NetBox IMHO.